File: src/main/java/org/kite9/diagram/visualization/orthogonalization/Orthogonalizer.java
package org.kite9.diagram.visualization.orthogonalization;

import org.kite9.diagram.visualization.planarization.Planarization;

/**
 * Creates the orthogonal representation from the Planarization.
 * 
 * @author robmoffat
 *
 */
public interface Orthogonalizer {

	public Orthogonalization createOrthogonalization(Planarization pln);
}

File: kite9-visualization/src/commonMain/kotlin/org/kite9/diagram/visualization/orthogonalization/Orthogonalizer.kt
package org.kite9.diagram.visualization.orthogonalization

import org.kite9.diagram.visualization.planarization.Planarization

/**
 * Creates the orthogonal representation from the Planarization.
 *
 * @author robmoffat
 */
interface Orthogonalizer {

    fun createOrthogonalization(pln: Planarization): Orthogonalization
}